test/functional/page_objects/visual_builder_page.ts

```diff
@@ -156,6 +156,35 @@ export function VisualBuilderPageProvider({ getService, getPageObjects }: FtrPro
       await element.click();
     }
 
+    /**
+     * setting label for markdown visualization
+     *
+     * @param {string} variableName
+     * @param type
+     * @memberof VisualBuilderPage
+     */
+    public async setMarkdownDataVariable(variableName: string, type: 'variable' | 'label') {
+      const SELECTOR = type === 'label' ? '[placeholder="Label"]' : '[placeholder="Variable name"]';
+      const prevRenderingCount = await PageObjects.visualize.getVisualizationRenderingCount();
+      if (variableName) {
+        await find.setValue(SELECTOR, variableName);
+      } else {
+        const input = await find.byCssSelector(SELECTOR);
+        if (process.platform === 'darwin') {
+          // Mac workaround
+          for (let i = 0; i <= type.length; i++) {
+            await input.pressKeys(browser.keys.BACK_SPACE);
+          }
+        } else {
+          await input.pressKeys([browser.keys.CONTROL, 'a']); // Select all for everything else
+          await input.pressKeys(browser.keys.NULL); // Release modifier keys
+          await input.pressKeys(browser.keys.BACK_SPACE); // Delete all content
+        }
+      }
+      await PageObjects.visualize.waitForVisualizationRenderingStabilized();
+      await PageObjects.visualize.waitForRenderingCount(prevRenderingCount + 1);
+    }
```

Comment:
I doubt there is a need to have both:
```js
const prevRenderingCount = await PageObjects.visualize.getVisualizationRenderingCount();
await PageObjects.visualize.waitForRenderingCount(prevRenderingCount + 1);
```
and
```js
await PageObjects.visualize.waitForVisualizationRenderingStabilized();
```

Usually we use one of it:
1) number of renderings is constant, e.g. +2 after some action
2) number of renderings is changing, we wait for it been fixed within 1 second